lib/account-data.js

```diff
@@ -1,95 +1,159 @@
 const fs = require('fs')
 const path = require('path')
 const rimraf = require('rimraf')
-const os = require("os");
-const {data,appDataPath} = require('./config')
-const userFiles = path.join(os.homedir(),'AppData','Roaming')
-const parsecUser = path.join(userFiles,'Parsec','user.bin')
-const accountData = path.join(appDataPath,'account_data.json')
-const currentAccountFile = path.join(appDataPath,'currentAccount.txt')
-
-const multiUserFolder = path.join(userFiles,'Parsec','UserList')
-
+const {global_state,initialize} = require("./initialize")
 
+const {fileFuncs} = require("./file-operations")
+// initialize()
 //todo need some rewriting.
 
-
 function checkNickname(nickname){
+
     let exist = 0
-    if(data.has(nickname)){
+    if(global_state.data.has(nickname)){
         exist = 1
     }
     return exist
 }
 
-function removeParsecUserFile(){
-    if (fs.existsSync(parsecUser)){
-        fs.unlinkSync(parsecUser)
+async function removeParsecUserFile(){
+    if (fs.existsSync(global_state.parsecUser)){
```

Comment:
use async/await alternative if that exists, it'll be better for GUI

Reply:
done